### PR TITLE
Videos

### PR DESCRIPTION
**Pull request by ModernTek**
This is initial release of videos part. I have used static images loaded from internet so far to see how it looks like. The next version I am going to use JS to load images, because those videos never will be the same. Please let me know if the solution is acceptable.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -224,7 +224,33 @@ <h2>im the main section</h2>
...
     <section class="developer-videos">
-      <h2>im developer videos section</h2>
+      <h1 class='developer-videos__title'>Watch Developers at Work and Play - Live</h1>
+      <div class="developer-videos-wrapper">
+        <figure class="developer-videos__figure">
+          <h1 class="developer-videos__location">Stocholm, Sweden</h1>
+          <h2 class="developer-videos__name">Mark Abrahamson</h2>
+          <img class="developer-videos__img" src="http://500px.com/graphics/pages/team/squares/oleg.jpg">
+        </figure>
+
```
Comment (Gruximillian) [inline src/index.html]: No need for a blank line here.

File: src/styles/layout/_developerVideos.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  width: 100%;
+  min-height: 200px;
+  height: auto;
+  margin: 0 auto;
+  background-color: white;
+  box-sizing: border-box;
+  padding: 0;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-align-items: center;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: I believe that webpack adds vendor prefixes, so you don't need to bother with those. You can check that if you remove them, and if the still appear in the page inspector.

File: src/styles/layout/_developerVideos.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  justify-content: center;
+  -webkit-flex-direction: row;
+  flex-direction: row;
+  -webkit-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-align-content: flex-end;
+  align-content: flex-end;
+ }   
+ .developer-videos__img {
+   border:0;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Please add a space after the colon.

File: src/styles/layout/_developerVideos.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+   width:100%;
+   height:100%;
+}
+.developer-videos__figure {
+  width: 25em;
+  height: 25em;
+  display:block;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Space after the colon.

File: src/styles/layout/_developerVideos.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
+.developer-videos {
+  position: relative;
+  background: none;
+  padding-bottom: 1em;
+  font-size: 1.0rem;
+  width: 100%;
+}
+.developer-videos-wrapper{
+  max-width: 96rem;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Add space between the selector and the opening brace.

File: src/styles/layout/_developerVideos.scss [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+.developer-videos__location, .developer-videos__name {
+  text-align: left;
+  padding-bottom: 10px;
+  text-transform: capitalize;
+  margin: 0;
+  padding-right: 1.3em;
+}
+
+.developer-videos__name{
+  color:red;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Add spaces before brace and after the colons.

File: src/styles/layout/_developerVideos.scss [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  text-transform: capitalize;
+  text-align: center;
+}
+
+.developer-videos__load-more{
+  //padding: 0px 40px;
+  margin-top: 0;
+  margin-bottom: 3em;
+  text-align: center;
+  font-size:  2em;
```
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Replace two spaces after the colon with one.

**Comment by ModernTek** [comment]
OK. Let me try to do video.

Thanks for comment

**Comment by Gruximillian** [comment]
@ModernTek 
Check the spaces on all of your css and try to make then consistent. It's a small thing, but goes a long way in making the code clean.

File: src/styles/layout/_developerVideos.scss (hunk 6 above)
Comment (Gruximillian) [inline src/styles/layout/_developerVideos.scss]: Almost forgot about this one. Use the `$red` variable and you'll get the correct color for the names.

**Comment by Gruximillian** [comment]
Ooops, sorry, I closed it accidentally.

**Comment by ModernTek** [comment]
I have added posters and UI custom video controls. I have added two images to index.js. @OscarRobertRodriguez and @Gruximillian please review and let me know Mean time I will write JS for custom video controls

**Comment by ModernTek** [comment]
I have added posters and UI for custom video controls. @Gruximillian @OscarRobertRodriguez please review. I will add JS for custom video controls afterwards.

**Comment by Gruximillian** [comment]
@ModernTek 
Posters look nice, and the code is good. But the controls need more styling, the alignment, the appearance, colors, all need a bit more work. You could maybe use that `$red` color for the controls to fit it in the site styles, or mabye a bit darker, you can use sass function `darken($red, 25%)` to get a some percentage darker color.

**Comment by ModernTek** [comment]
Vojislav,

I will work more on styling. First I would like to finish with JS.

Thank you for comments

**Pull request merged**